starknet_py/tests/e2e/fixtures/contracts.py

```diff
@@ -29,6 +29,17 @@ def map_compiled_contract() -> str:
     return read_contract("map_compiled.json")
 
 
+@pytest.fixture(scope="package")
+def sierra_minimal_compiled_contract_and_class_hash() -> Tuple[str, int]:
+    """
+    Returns minimal contract compiled to sierra
+    """
+    return (
+        read_contract("precompiled/minimal_contract_compiled.json"),
+        0x3FD6D06A297540CA6F317B2260C3ABD6B616C961D820DEA2800EBC34E6336A4,
```

Comment:
It is computed by hand if I understand correctly, right?

Reply:
Yes, computed through a metod from cairo-lang

Reply:
Which method exactly was used to compute it?

Reply:
```py
from starkware.starknet.core.os.contract_class.compiled_class_hash import (
    compute_compiled_class_hash,
)
```